.github/workflows/pre-release.yml

```diff
@@ -77,6 +77,95 @@ jobs:
           CARGO_REGISTRIES_CRATES_IO_PROTOCOL: sparse
           ASYNC_STD_THREAD_COUNT: 4
 
+  checks-ubuntu-24:
+    name: Code checks on Ubuntu 24.04
+    runs-on: ubuntu-latest
+    container: ubuntu:24.04
+    steps:
+      - name: Update and install packages
+        run: apt-get update && apt-get install -y curl git build-essential
+
+      - name: Install Rust toolchain
+        run: curl --proto '=https' --tlsv1.2 -sSf https://sh.rustup.rs | sh -s -- -y
+
+      - name: Clone this repository
+        uses: actions/checkout@v4
+
+      - name: Add Zenoh Rust toolchain version
+        run: |
+          . "$HOME/.cargo/env"
+          rustup override set $(cat rust-toolchain.toml | grep '^channel' | sed 's/[^"]*"//' | sed 's/"//')
```

Comment:
Is this really necessary? Won't `rustup` install the toolchain from the file by itself?

Reply:
I remember testing and noticing that it does not install it. But I just tested again and it works 😅